src/components/clusterConfiguration/clusterValidations.ts

```diff
@@ -49,9 +49,18 @@ export const validateRequiredFields = (cluster: Cluster) => {
   try {
     validateYupSchema<ClusterConfigurationValues>(values, installValidationSchema, true);
   } catch (err) {
-    const errors = yupToFormErrors(err);
-    const errorFields = Object.keys(errors).map((field: string) => CLUSTER_FIELD_LABELS[field]);
-    return `Not all required cluster properties are configured yet: ${errorFields.join(', ')}.`;
+    if (err.name === 'ValidationError') {
```

Comment:
instanceof ValidationError would be a safer approach as we wont have to use string
```suggestion
    if (err instanceof ValidationError) {
```

Reply:
Done